src/store/modules/console.js

```diff
@@ -34,6 +35,7 @@ const mutations = {
       layer.selected = i === index;
       layer.legend_displayed = i === index || i === 0; // always display the first legend
     });
+    store.commit("timelapse/displayTimelapse");
```

Comment:
Same thing as previous comment- do we want the time-lapse component to disappear if the user has not selected a sensor and switches back from the inundation to the sensors layer?